src/backend/templates/rooms.hbs

```diff
@@ -120,14 +120,14 @@
 		             	    </p>
 		             	  {{/if}}
 		             	{{/speakers_list}}
-		             	<ul style="padding:0;font-size:15px;">
+                    <a href="javascript:;"><ul style="padding:0;font-size:15px;">
```

Comment:
@aayusharora I was trying to add a dummy link to each box as tabs can reach only those items which have some link in them